packages/storage/amplify_storage_s3/example/integration_test/main_test.dart

```diff
@@ -446,6 +448,27 @@ void main() async {
           smallFileBytes: testBytes,
           largeFileBytes: testLargeFileBytes,
         );
+
+        testTransferAcceleration(
+          dataPayloads: [
+            Tuple4(
```

Comment:
Can you create a class instead? Sorry 😅 In Dart 3, we'll get records, but until then I can only guess at what these params mean

Reply:
Updated